.github/workflows/changed_solutions.yml

```diff
@@ -0,0 +1,61 @@
+name: Changed Solutions
```

Comment:
My hope was that this change would augment the contents of the existing `solutions.yml`. Ideally, as long as this action tests any changed solutions, we don't need to wait for a full re-run of all solutions.